.gitignore

```diff
@@ -4,11 +4,9 @@ composer.lock
 script/*
 
 # Testing
-/phpunit.xml
 /build
 /test/reports
 .env
 .idea
 /.php_cs
-/.php_cs.cache
-/.phpunit.result.cache
+*.cache
```

Comment:
should we use `/*.cache` just to be safe?

Reply:
Yep, let me change it.